app_dart/lib/src/request_handlers/github_rate_limit_status.dart

```diff
@@ -29,7 +29,7 @@ class GithubRateLimitStatus extends RequestHandler<Body> {
 
   @override
   Future<Body> get() async {
-    final GithubService githubService = await config.createGithubService('flutter', 'flutter');
+    final GithubService githubService = await config.createGithubService(config.flutterSlug);
```

Comment:
Do we check this only for the flutter repo?

Reply:
The quota is the same across all Flutter repos because they all share the flutter-dashboard app. `packlage If operations like this become more common, I created a `config.createDefaultGithubService()` to hide this detail.